roles/sap_swpm/tasks/pre_install.yml

```diff
@@ -81,3 +74,32 @@
       - '   The installation can take up to 3 hours. Run the following command as root'
       - '   on {{ ansible_hostname }} to display the installation logs:'
       - '   # tail -f $(cat /tmp/sapinst_instdir/.lastInstallationLocation)/sapinst.log'
+
+################
+# Set sapinst command
+################
+
+- name: SAP SWPM Pre Install - Set the sapinst command parameters
+  vars:
+    sap_swpm_swpm_command_guiserver: "{{ 'SAPINST_START_GUISERVER=false' if not sap_swpm_swpm_observer_mode else '' }}"
+    sap_swpm_swpm_command_observer: "{{ 'SAPINST_REMOTE_ACCESS_USER=' + sap_swpm_swpm_remote_access_user + ' SAPINST_REMOTE_ACCESS_USER_IS_TRUSTED=true' if sap_swpm_swpm_observer_mode and sap_swpm_swpm_remote_access_user is defined and sap_swpm_swpm_remote_access_user | length > 0 else '' }}"
```

Comment:
There will be merge conflict with https://github.com/sap-linuxlab/community.sap_install/pull/850/files